lib/rules/no-unnecessary-act.ts

```diff
@@ -32,37 +33,71 @@ export default createTestingLibraryRule<[], MessageIds>({
         'Avoid wrapping Testing Library util calls in `act`',
       noUnnecessaryActEmptyFunction: 'Avoid wrapping empty function in `act`',
     },
-    schema: [],
+    schema: [
```

Comment:
Added it for consistency, but not sure why `schema` is needed and how and where it's used. Can someone enlighten me? 😄

Reply:
You can find ESLint Options Schemas doc here: https://eslint.org/docs/developer-guide/working-with-rules#options-schemas

TL;DR ESLint will use that schema to validate configuration options and prevent invalid or unexpected inputs before  executing the rules:

<img width="1215" alt="ESLint config validation error displayed based on rule schema" src="https://user-images.githubusercontent.com/2677072/124609138-a7a0c700-de6f-11eb-8732-469eb476f723.png">

Reply:
I just realized the option for `no-dom-import` is specified just as a `string` in the schema, but can be improved to indicate only `"angular"`, `"react"`, or `"vue"` are the valid options.